api-auto-registration/about.hbs.md

```diff
@@ -2,4 +2,19 @@
 
 ## <a id='overview'></a> Overview
 
-API Auto Registration automates the registration of API specification defined in a workload's configuration. The registered API specification will then be accessible in TAP GUI without any additional manual steps. API Auto Registration is leveraged by an automated workflow using a supply chain to create and manage a Kubernetes Custom Resource (CR) of kind `APIDescriptor`. A Kubernetes controller will reconcile the CR and update the API entity in TAP GUI to achieve automated API specification registration from origin workloads. You may also use API Auto Registration without supply chain automation by direction applying an `APIDescriptor` CR to the cluster.
+API Auto Registration automates the registration of API specification defined in a workload's configuration. The registered 
+API specification will then be accessible in TAP GUI without any additional manual steps. API Auto Registration is leveraged 
+by an automated workflow using a supply chain to create and manage a Kubernetes Custom Resource (CR) of kind `APIDescriptor`. 
+A Kubernetes controller will reconcile the CR and update the API entity in TAP GUI to achieve automated API specification 
+registration from origin workloads. You may also use API Auto Registration without supply chain automation by direction 
+applying an `APIDescriptor` CR to the cluster.
+
+## <a id='getting-started'></a> Getting Started
+
+Learn more about APIDescriptor CR and API entities in TAP GUI in the [Key Concepts section](key-concepts.md)
+
+For "iterate", "run" and "full" TAP cluster profiles, start at the [Usage section](usage.md)
+
+For other profiles, install the api-auto-registration package using the [Installation section](installation.md)
```

Comment:
I see, you have a separate section in `usage` to talk about update. I'm fine with that option too.

Reply:
Yeah I debated repeating, but since the command for update was slightly different anyway and users have a focused page, I decided to go with the repeats. But I can check what other teams are doing

Reply:
Check my newer commit's usage page first paragraph